pkg/applicationserver/applicationserver.go

```diff
@@ -145,6 +141,11 @@ func New(c *component.Component, conf *Config) (as *ApplicationServer, err error
 		interopClient: interopCl,
 		interopID:     conf.Interop.ID,
 	}
+	as.formatters = messageprocessors.MapPayloadProcessor{
+		ttnpb.PayloadFormatter_FORMATTER_JAVASCRIPT: javascript.New(),
+		ttnpb.PayloadFormatter_FORMATTER_CAYENNELPP: cayennelpp.New(),
+		ttnpb.PayloadFormatter_FORMATTER_REPOSITORY: devicerepository.New(as.formatters, as),
```

Comment:
Is this line correct here instead of after initializing `as.formatters` as it was before? `as.formatters` is an empty map at this point when it's passed to `devicerepository.New`, right?

Reply:
Great catch!